lib/element.js

```diff
@@ -161,6 +161,100 @@ ElementArrayFinder.prototype.all = function(locator) {
 };
 
 /**
+ * Get a slice of elements within the ElementArrayFinder.
+ * Negative indices are wrapped (i.e. -i means ith element from last)
+ * This does not actually retrieve the underlying element.
+ *
+ * Slice can be negative in which case we go backwards.
+ *
+ * Note that if step is positive and start > stop, an error is thrown
```

Comment:
The slicing syntax is modeled after the slice syntax in python. It differs from Array.prototype.slice in the sense that it accepts an additional step parameter. elems.slice(2,-1) is valid and should produce the result as shown below : 

 * @view
 * <ul class="items">
 *   <li>Zeroth</li>
 *   <li>First</li>
 *   <li>Second</li>
 *   <li>Third</li>
 *   <li>Fourth</li>
 *   <li>Fifth</li>
 *   <li>Sixth</li>
 *   <li>Seventh</li>
 * </ul>
 *
expect(element.all(locator).slice(2,-1).getText()).toEqual( ['Second','Third','Fourth','Fifth','Sixth']);

I am wondering if I should model this differently or stick to my current implementation and perhaps rename the method so as to not confuse it with Array.prototype.slice ?